extensions/wikia/UserLogin/UserSignupSpecialController.class.php

```diff
@@ -460,28 +460,26 @@ private function isWithinRegistrationPerEmailLimit( $email ) {
 	}
 
 	/**
-	 * @desc Checks if the user/email is not blocked in phalanx and sets the proper response if not
+	 * @desc Checks if the user/email is blocked in phalanx and sets the proper response if so
 	 *
 	 * @param $user
 	 * @return bool
 	 */
-	private function isNotBlockedByPhalanx( $user ) {
+	private function isBlockedByPhalanx( $user ) {
+		$abortError = '';
 
-		return UserLoginHelper::callWithCaptchaDisabled(function($params) {
```

Comment:
Yea, I realize that blithely saying 'This code runs fine without wrapping it' doesn't inspire much confidence :) The reason why we don't need ```callWithCaptchaDisabled``` anymore is because one one of the handlers that used to get run was trying to verify the captcha. Now that we're only calling the ```PhalanxUserBlock::abortNewAccount``` handler we no longer need to worry about captcha verification. See my comment below for more info.